portal/static/portal/sass/styles.scss

```diff
@@ -1148,6 +1203,74 @@ td {
   }
 }
 
+@media (max-width: 1570px) {
+
+  .onboarding__step {
+    height: 76px;
+
+    img {
+      @include _margin(0px, 10px, 5px, 10px);
+    }
+
+    p {
+      @include _line-height(20px);
+      @include _margin(0px, 0px, 0px, 0px);
+    }
+
+  }
+
+  .onboarding__step__text {
+    @include _margin(0px, 0px, 0px, 45px);
+    z-index: 1;
+  }
+
+  .onboarding__step--done {
+
+    p {
+      @include _margin(0px, -10px, 0px, 0px);
+    }
+
+  }
+
+  .onboarding__step--done:before {
+    height: 53px;
+    width: 53px;
+  }
+
+  .onboarding__step--done:after {
+    height: 53px;
+    right: -53px;
+    width: 53px;
+  }
+
+  .onboarding__step--in-progress:before {
+    border-bottom: 38px solid transparent;
```

Comment:
You've already set the style and colour, so I think what you really want is `border-bottom-width`.

Also, make the 38 a var.  Is it related to the 53?